bundles/org.openhab.automation.jsscripting/README.md

```diff
@@ -21,23 +36,215 @@ log:set DEBUG org.openhab.core.automation
 
 For more information on the available APIs in scripts see the [JSR223 Scripting]({{base}}/configuration/jsr223.html) documentation.
 
-## Script Examples
+The following examples show how to access common openHAB functionalities.
+
+## Logging
 
-JavaScript scripts provide access to almost all the functionality in an openHAB runtime environment.
 As a simple example, the following script logs "Hello, World!".
 Note that `console.log` will usually not work since the output has no terminal to display the text.
+__Please note:__ Support for `console.log` will likely be added together with a logging API in the [helper library](https://github.com/openhab/openhab-js).
```

Comment:
Actually this will not be provided by the helper library, but will be maintained as part of the JSScripting binding itself very soon.  See https://github.com/digitaldan/openhab-addons/blob/f99e8affce197a95d4b829b31b1907cefce10819/bundles/org.openhab.automation.jsscripting/src/main/resources/node_modules/@globals.js